SamplesDashboard/SamplesDashboard/ClientApp/src/views/Details.tsx

```diff
@@ -148,34 +184,42 @@ function renderItemColumn(item: IDetailsItem, index: number | undefined, column:
         case 'Library':
             return <span>{packageName} </span>;
 
-        case 'Used Version':
+        case 'Used version':
             return <span>{requirements} </span>;
 
-        case 'Latest Version':
+        case 'Latest version':
             return <span>{currentVersion} </span>;
 
-        case 'Azure SDK Version':
+        case 'Azure SDK version':
```

Comment:
The corresponding collumn name in line 64 has not been renamed. This is causing the azureSDKVersion to not display on the UI

Reply:
Thank you, will resolve this